ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -573,6 +729,59 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-proxy-fips-minimal-permission-f28
+  cluster: build01
+  cron: 32 11 8 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-nutanix-ipi-proxy-minimal-permission
+- as: nutanix-ipi-zones-f28
+  cluster: build01
+  cron: 6 7 10 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-nutanix-ipi-zones
+- as: vsphere-8-ipi-ovn-etcd-encryption-f28
```

Comment:
The main test point for the install workflow is to cover `vsphere-8`, we already the vsphere platform is already upgraded to version 8, and we also have another similar upgrade ci jobs - `vsphere-ipi-compact-etcd-encryption-f28`, so do not need to run this job.